### PR TITLE
INT-9 (Navbar enhancement)

### PR DESCRIPTION
**Pull request by cchanyu**
# Notes:
- Displayed HBClogo into the navbar
- Installed/Imported font awesome components
- Icon display and box set up complete
- Icon response to clicks

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import Dropdown from './components/Dropdown/Dropdown';
 import { WaitingSpinner } from '@hbc/hbc-core-components';
 //import { Card } from '@hbc/hbc-core-components';
 
+import HBClogo from './images/hbctech_logo.jpg';
+
```
Comment (richsong) [inline src/App.js]: remove this extra line

File: src/App.js [hunk 2]
```diff
@@ -44,7 +46,7 @@ this.setState({isLoading: true})
     
     return(
       <div className="app">
-        <Navbar />
+        <Navbar HBClogo={HBClogo} />
```
Comment (richsong) [inline src/App.js]: Rename this `HBClogo` prop to `logoPath`

File: src/components/navbar/navbar.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faUser } from '@fortawesome/free-solid-svg-icons'
+
 import './Navbar.scss'
-import Home from '../Home/Home';
 
-//import Sakslogo from '../../images/01_saksfifth.jpg';
+library.add(faUser);
```
Comment (richsong) [inline src/components/navbar/navbar.js]: do we need this to get FA to work??

Comment (cchanyu) [reply src/components/navbar/navbar.js]: yes, apparently. the library is necessary.

File: src/components/navbar/navbar.js [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
 
 class Navbar extends Component {
```
Comment (richsong) [inline src/components/navbar/navbar.js]: It looks like we are starting to add a number of props to this component. Please add props validation to it now.
https://reactjs.org/docs/typechecking-with-proptypes.html

File: src/App.js (hunk 1 above)
Comment (richsong) [reply src/App.js]: also rename HBClogo to HBClogoPath or something like that

File: src/App.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ import Dropdown from './components/Dropdown/Dropdown';
 
 import { WaitingSpinner } from '@hbc/hbc-core-components';
 //import { Card } from '@hbc/hbc-core-components';
-
+import HBClogo from './images/hbctech_logo.jpg';
```
Comment (richsong) [inline src/App.js]: I'm having second thoughts on this. If we are NOT going to be passing the image path from the JSON payload, let's import this directly into the NavBar component itself. In other words, let's remove this import statement in App.js and remove the logoPath props.

**Pull request merged**